src/Storage/Storage/Storage.csproj

```diff
@@ -23,11 +23,7 @@
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.2.2" />
     <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="11.2.2" />
   </ItemGroup>
-  <Target Name="CopyFiles" AfterTargets="Build">
-    <Copy SourceFiles="@(PreLoadAssemblies)" DestinationFolder="$(TargetDir)\PreloadAssemblies" />
-    <Copy SourceFiles="@(NetCoreAssemblies)" DestinationFolder="$(TargetDir)\NetCoreAssemblies" />
-  </Target>
```

Comment:
These lines were added to support data plane track 2 SDK (#11520). They preload Azure.Core.dll and its dependencies. But now Azure.Core is handled by Az.Accounts so these are no longer needed.
cc @blueww